core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1,32 +1,7 @@
 package io.aiven.klaw.helpers.db.rdbms;
 
 import com.google.common.collect.Lists;
-import io.aiven.klaw.dao.Acl;
-import io.aiven.klaw.dao.AclID;
-import io.aiven.klaw.dao.AclRequestID;
-import io.aiven.klaw.dao.AclRequests;
-import io.aiven.klaw.dao.ActivityLog;
-import io.aiven.klaw.dao.Env;
-import io.aiven.klaw.dao.EnvID;
-import io.aiven.klaw.dao.KafkaConnectorRequest;
-import io.aiven.klaw.dao.KafkaConnectorRequestID;
-import io.aiven.klaw.dao.KwClusterID;
-import io.aiven.klaw.dao.KwClusters;
-import io.aiven.klaw.dao.KwKafkaConnector;
-import io.aiven.klaw.dao.KwProperties;
-import io.aiven.klaw.dao.KwRolesPermissions;
-import io.aiven.klaw.dao.KwTenants;
-import io.aiven.klaw.dao.ProductDetails;
-import io.aiven.klaw.dao.RegisterUserInfo;
-import io.aiven.klaw.dao.SchemaRequest;
-import io.aiven.klaw.dao.SchemaRequestID;
-import io.aiven.klaw.dao.Team;
-import io.aiven.klaw.dao.TeamID;
-import io.aiven.klaw.dao.Topic;
-import io.aiven.klaw.dao.TopicID;
-import io.aiven.klaw.dao.TopicRequest;
-import io.aiven.klaw.dao.TopicRequestID;
-import io.aiven.klaw.dao.UserInfo;
+import io.aiven.klaw.dao.*;
 import io.aiven.klaw.model.enums.AclPatternType;
```

Comment:
Hey Intellij strikes and bound these together into a .*

Reply:
there are about 26 imports, and now in Intellij I have 20 max. I think 20 or 25 is ideal ?